docs/running-a-node/docker-management.md

```diff
@@ -140,6 +140,31 @@ Once modifications are complete, [start up the docker container again](#starting
 
 ---
 
+### Docker entrypoint support
+
+The docker entry script has migrated to a command with default arguments:
+```
+Usage:
+   /entry.sh nano_node [daemon] [cli_options] [-l] [-v size]
```

Comment:
@argakiig There was some confusion on Discord about having the `daemon` get passed in naked instead of as `--daemon` as it is done elsewhere: https://docs.nano.org/commands/command-line-interface/#-daemon - was this done to avoid confusion if other cli_options were passed without the `--daemon` and that isn't supported?

Reply:
either way works TBH it will pass the -- through properly